libs/dh/shared/data-access-msw/src/lib/mockedServiceWorker.ts

```diff
@@ -17,8 +17,10 @@
 import { setupWorker } from 'msw';
 import { handlers, onUnhandledRequest } from './handlers';
 
-try {
-  const worker = setupWorker(...handlers);
-  worker.start({ onUnhandledRequest });
-  // eslint-disable-next-line no-empty
-} catch (error) {}
+export function setupServiceWorker(apiBase: string) {
+  try {
+    const worker = setupWorker(...handlers(apiBase));
+    worker.start({ onUnhandledRequest });
+    // eslint-disable-next-line no-empty
+  } catch (error) {}
+}
```

Comment:
nitpick: can we name the file `setup-service-worker.ts`. To reflect the function's name, but also for consistency with general file naming. Can see that some files don't follow that but let's handle that in separate PRs.

Reply:
Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1023/commits/292a3d1e48691ab462bca94911c520779f4cbe91